modules/mono/editor/GodotTools/GodotTools.ProjectEditor/GodotTools.ProjectEditor.csproj

```diff
@@ -31,14 +31,7 @@
     <Reference Include="System" />
     <Reference Include="Microsoft.Build" />
     <Reference Include="DotNet.Glob, Version=2.1.1.0, Culture=neutral, PublicKeyToken=b68cc888b4f632d1, processorArchitecture=MSIL">
-      <!--
-      When building Godot with 'mono_glue=no' SCons will build this project alone instead of the
-      entire solution. $(SolutionDir) is not defined in that case, so we need to workaround that.
-      We make SCons restore the NuGet packages in the project directory instead in this case.
-      -->
-      <HintPath Condition=" '$(SolutionDir)' != '' ">$(SolutionDir)\packages\DotNet.Glob.2.1.1\lib\net45\DotNet.Glob.dll</HintPath>
```

Comment:
This is needed when building the entire solution instead of a specific project. What we should do here is add conditions:

```csproj
<HintPath Condition=" '$(SolutionDir)' != '' And Exists('$(SolutionDir)\packages\...') ">$(SolutionDir)\packages\...</HintPath>
<HintPath Condition=" '$(SolutionDir)' == '' Or !Exists('$(SolutionDir)\packages\...') ">$(ProjectDir)\packages\...</HintPath>
```

Reply:
Only GodotTools.ProjectEditor uses DotNet and has a packages.config for it. It isn't a package used for the whole solution.

Reply:
But when restoring the entire solution, the packages folder is created in the solution directory instead of the project directory. At least when running `nuget restore`, which is what SCons uses; I didn't check with `msbuild /t:restore`.

Reply:
It happens to me too. The solution restores a DotNet package which is not used for any project and GodotTools.ProjectEditor another DotNet specific for it. They are even different versions. You can see details in the Manage NuGet panel in VS for the solution and the project.

Reply:
This is still pending. I just checked again to be sure. When running `nuget restore` on a solution, the `packages` folder is created in the solution dir. The alternative hintpaths I commented above should always work.